### PR TITLE
Quiz management

### PR DESCRIPTION
**Pull request by chanchal-panpaliya**
live link -> https://genuine-quokka-d805c9.netlify.app/

provide your comments

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET=reactapp
```
Comment (ttahm3d) [inline .env:1]: Exclude `.env` files from your git

File: src/web-app/Pages/LeaderBord/Leaderboard.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import Header from 'web-app/Component/Header/Header';
+import { useTheme } from "../../Context/theme-context";
+import QuizContext from 'web-app/Context/QuizContext';
+import { useContext ,useEffect ,useState} from 'react';
+
+
+
+const Leaderboard=()=>{
+    const { darkTheme } = useTheme();
+    let {leaderlist} = useContext(QuizContext);
```
Comment (ttahm3d) [inline src/web-app/Pages/LeaderBord/Leaderboard.jsx:12]: can be simplified like you had done for useTheme

File: src/web-app/Pages/LeaderBord/Leaderboard.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    const [data,setdata] = useState([])
+
+    useEffect(()=>{
+       let sort =  leaderlist.sort((a,b)=>{
+           return b.score - a.score
+       })
+       console.log(sort)
```
Comment (ttahm3d) [inline src/web-app/Pages/LeaderBord/Leaderboard.jsx:19]: Remove console log

File: src/web-app/Pages/Result/Result.jsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+import { Link ,useParams,useNavigate} from 'react-router-dom';
+import Header from 'web-app/Component/Header/Header';
+import Footer from 'web-app/Component/Footer/Footer';
+import { useTheme } from "../../Context/theme-context";
+import { Modal } from 'web-app/Component/Modal/Modal';
+
+const Result =()=>{
+    const { darkTheme } = useTheme();
+    const {id} = useParams();
+    let {leaderlist,quizresult,GetSingleQuizId} = useContext(QuizContext)
```
Comment (ttahm3d) [inline src/web-app/Pages/Result/Result.jsx:13]: What are using let if the values ar not changing

**Review by ttahm3d — COMMENTED** [review]
Have added few comments please take look.

File: .env (hunk 1 above)
Comment (chanchal-panpaliya) [reply .env:1]: I forgot to add the .env file, so I added the .env file now.

File: src/web-app/Pages/LeaderBord/Leaderboard.jsx (hunk 3 above)
Comment (chanchal-panpaliya) [reply src/web-app/Pages/LeaderBord/Leaderboard.jsx:19]: done

**Review by anshtripathi01 — APPROVED** [review]
you can make even better footer otherwise UI and code looks clean and good to merge

**Pull request merged**